webapp/components/features/FollowList/FollowList.spec.js

```diff
@@ -0,0 +1,129 @@
+import { config, mount } from '@vue/test-utils'
+import Vuex from 'vuex'
+
+import helpers from '~/storybook/helpers'
+import FollowList from './FollowList.vue'
+
+const localVue = global.localVue
+
+config.stubs['client-only'] = '<span><slot /></span>'
+config.stubs['ds-space'] = '<span><slot /></span>'
+config.stubs['nuxt-link'] = '<span><slot /></span>'
+
+describe('FollowList.vue', () => {
+  let store, mocks, getters, propsData
+
+  beforeAll(() => {
+    mocks = {
+      $t: jest.fn(),
+    }
+    getters = {
+      'auth/user': () => {
+        return {}
+      },
+      'auth/isModerator': () => false,
+    }
+    const [_user] = helpers.fakeUser()
+    propsData = {
+      user: {
+        ..._user,
+        followedByCount: 12,
+        followingCount: 15,
+        followedBy: helpers.fakeUser(7),
+        following: helpers.fakeUser(7),
+      },
+    }
+  })
+
+  describe('mount', () => {
+    beforeAll(() => {
+      store = new Vuex.Store({
+        getters,
+      })
+    })
+
+    describe('given a user with connections', () => {
+      ;['following', 'followedBy'].forEach((type) =>
+        describe(`and type=${type}`, () => {
+          let wrapper
+          let queryMock
+
+          beforeAll(() => {
+            queryMock = jest.fn().mockResolvedValue({
+              data: { User: [{ [type]: additionalConnections[type] }] },
+            })
+
+            wrapper = mount(FollowList, {
+              store,
+              propsData: { ...propsData, type: type },
+              mocks: {
+                ...mocks,
+                $apollo: {
+                  query: queryMock,
+                },
+              },
+              localVue,
+            })
+          })
+
+          it(`shows the users ${type}`, () => {
+            expect(wrapper.findAll('.user-teaser').length).toEqual(propsData.user[type].length)
+          })
+
+          it(`has a button to load all remaining users ${type}`, async () => {
+            jest.useFakeTimers()
+
+            wrapper.find('button').trigger('click')
+            await jest.runAllTicks()
+            await wrapper.vm.$nextTick()
+
+            expect(wrapper.vm.connections.length).toBe(propsData.user[`${type}Count`])
+            expect(queryMock).toHaveBeenCalledWith({
+              query: wrapper.vm.queries[type],
+              variables: { id: propsData.user.id },
```

Comment:
removed, because not needed anymore in [e6361c9](https://github.com/Human-Connection/Human-Connection/pull/3293/commits/e6361c9b740f104c0a0d86b463be835f10311f3d#diff-970486bf70217f927b18aadea0c8b1b4)

In this case would have been necessary, since the user is generated by `faker`, i.e. the id not known before the test runs.